qiskit_ibm_provider/utils/backend_converter.py

```diff
@@ -0,0 +1,150 @@
+        # Create measurement instructions:
+        measure_props = {}
+        for qubit, _ in enumerate(properties.qubits):
+            measure_props[(qubit,)] = InstructionProperties(
+                duration=properties.readout_length(qubit),
+                error=properties.readout_error(qubit),
+            )
+        target.add_instruction(Measure(), measure_props)
+    # Parse from configuration because properties doesn't exist
+    else:
+        for gate in configuration.gates:
+            name = gate.name
+            gate_props = (
+                {tuple(x): None for x in gate.coupling_map}  # type: ignore[misc]
+                if hasattr(gate, "coupling_map")
+                else {None: None}
+            )
+            gate_len = len(gate.coupling_map[0]) if hasattr(gate, "coupling_map") else 0
+            if name in name_mapping:
+                target.add_instruction(name_mapping[name], gate_props)
+            else:
+                custom_gate = Gate(name, gate_len, [])
+                target.add_instruction(custom_gate, gate_props)
+        target.add_instruction(Measure())
+    # parse global configuration properties
+    if hasattr(configuration, "dt"):
+        target.dt = configuration.dt
+    if hasattr(configuration, "timing_constraints"):
+        target.granularity = configuration.timing_constraints.get("granularity")
+        target.min_length = configuration.timing_constraints.get("min_length")
+        target.pulse_alignment = configuration.timing_constraints.get("pulse_alignment")
+        target.aquire_alignment = configuration.timing_constraints.get(
+            "acquire_alignment"
+        )
+    # If a pulse defaults exists use that as the source of truth
+    if defaults is not None:
+        inst_map = defaults.instruction_schedule_map
+        for inst in inst_map.instructions:
+            for qarg in inst_map.qubits_with_instruction(inst):
+                sched = inst_map.get(inst, qarg)
+                if inst in target:
+                    try:
+                        qarg = tuple(qarg)
+                    except TypeError:
+                        qarg = (qarg,)
+                    if inst == "measure":
+                        for qubit in qarg:
+                            target[inst][(qubit,)].calibration = sched
+                    else:
+                        target[inst][qarg].calibration = sched
+    return target
+
+
+def qubit_props_dict_from_props(
+    properties: BackendProperties,
+) -> Dict[int, IBMQubitProperties]:
+    """Uses BackendProperties to construct
+    and return IBMQubitProperties class.
+    """
+    qubit_props = {}
+    for qubit, _ in enumerate(properties.qubits):
+        qubit_props[qubit] = IBMQubitProperties(  # type: ignore[no-untyped-call]
+            t1=properties.t1(qubit),
+            t2=properties.t2(qubit),
+            frequency=properties.frequency(qubit),
+            anharmonicity=properties.qubit_property(qubit, "anharmonicity"),
```

Comment:
This value is reported in units of Hz if this convention is not updated with V2 migration. Same for frequency.

Reply:
Fixed! https://github.com/Qiskit/qiskit-ibm-provider/pull/317/commits/3e725797245fa08423252bc1456fca511b2fcd99